src/components/main/friends/friend/mod.rs

```diff
@@ -1,14 +1,18 @@
-use dioxus::prelude::*;
+use dioxus::{core::to_owned, prelude::*};
 use dioxus_heroicons::outline::Shape;
 use ui_kit::{
     activity_indicator::ActivityIndicator,
     button::Button,
     profile_picture::PFP,
     skeletons::{inline::InlineSkeleton, pfp::PFPSkeleton},
 };
-use warp::crypto::DID;
+use warp::{crypto::DID, raygun::Conversation};
```

Comment:
<img width="840" alt="Captura de ecrã 2022-12-05, às 18 57 14" src="https://user-images.githubusercontent.com/29093946/205720093-9cefb008-5fb9-4206-a41c-d755ab97c6dc.png">

Reply:
I can resolve the other warnings, but what about the `non_snake_case` and the props' "`unused_variables`"? May I ignore those?

Reply:
@sdwoodbury what do you think?

Reply:
@drepram you could ignore those by doing `#[allow(non_snake_case)]` on to of `Friends` and probably the same for the unused variables. Can also remove the mut there if its not needed for `rg`

Reply:
You are right!